templates/taxi/car_list.html

```diff
@@ -1,22 +1,75 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
   <h1>
-    Car list
+    Car List
     <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
       Create
     </a>
   </h1>
+  
+  <form method="get" action="" class="form-inline">
+  {{ search_form|crispy }}
+  <input type="submit" value="🔎" class="btn btn-secondary">
+  </form>
+  
+  
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th>Drivers</th>
+        <th>Update car</th>
+        <th>Delete car</th>
+        <th>Assign / Delete</th>
+      </tr>
+
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+              <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.model }}</a>
+          </td>
+          <td>
+              {{ car.manufacturer }}
+          </td>
+          <td>
+              <ul>
+                {% for driver in car.drivers.all %}
+                  <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+                {% endfor %}
+              </ul>
+          </td>
+          <td>
+              <a href="{% url 'taxi:car-update' pk=car.id %}">
+                Update
+              </a>
+            </td>
+            <td>
+              <a style="color: red"
+                href="{% url 'taxi:car-delete' pk=car.id %}">
+                Delete
+              </a>
+            </td>
+            <td>
+              {% if car in user.cars.all %}
+                <a href="{% url 'taxi:toggle-car-assign' pk=car.id %}" class="btn btn-danger">
+                  Delete me from this car
+                </a>
+              {% else %}
+                <a href="{% url 'taxi:toggle-car-assign' pk=car.id %}" class="btn btn-success">
+                  Assign me from this car
+                </a>
+              {% endif %}
+            </td>
+        </tr>
       {% endfor %}
-    </ul>
+    </table>
+
   {% else %}
-    <p>There are no cars in taxi</p>
+      <p>There are no manufacturers in the service.</p>
```

Comment:
The message 'There are no manufacturers in the service.' should be updated to 'There are no cars in the service.' to accurately reflect the context of the car list.